app/models/user.rb

```diff
@@ -1,4 +1,9 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
`:confirmable` should be enabled as well
```suggestion
   # :lockable, :timeoutable, :trackable and :omniauthable
   
   
   :confirmable, 
```